theano/tensor/nnet/abstract_conv.py

```diff
@@ -2705,9 +2830,9 @@ def correct_for_groups(mat):
             if self.filter_flip:
                 img = img[flip_filters]
 
-        if any(p > 0 for p in pad):
+        if any(p != (0, 0) or p != 0 for p in pad):
```

Comment:
This is a nice check that you could also do in the other `perform`s (instead of `if isinstance(pad, tuple)`).